llap-server/src/java/org/apache/hadoop/hive/llap/shufflehandler/ShuffleHandler.java

```diff
@@ -204,6 +204,9 @@ public class ShuffleHandler implements AttemptRegistrationListener {
 
   public static final String CONNECTION_CLOSE = "close";
 
+  public static final String SHUFFLE_SSL_ENABLED_KEY = "hive.llap.shuffle.ssl.enabled";
```

Comment:
to be consistent with other configs, should we name it simply `llap.shuffle.ssl.enabled`?